source/hic_hal/stm32/stm32h743ii/extended_features/include/adapter.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _ADAPTER_H_INCLUDED_
+#define _ADAPTER_H_INCLUDED_
+
+typedef enum
+{
+    ADAPTER_UDB_13_FLEX,
```

Comment:
Should there be an ADAPTER_NONE value? Having nothing attached and having an unknown adapter attached are two different cases.